### ISSUE TITLE
Missing rows in table users after updating

### ISSUE THREAD

**Issue by instantflorian** [WBCE/WBCE_CMS#407]
When updating an 1.3.3 to 1.4, the following rows in the table (prefix)_users are missing:
signup_checksum
signup_timestamp
signup_timeout
signup_confirmcode
So these rows have to be created by update script, otherwise it's not possible to create new users in a 1.4 installation.

**Comment by mrbaseman** [comment]
perhaps we should create the tables if they don't exist explicitly in the update script to avoid problems like [this one](https://forum.wbce.org/viewtopic.php?id=2736&action=new) where the dbsessions table was not created for some reason during the update

**Comment by mrbaseman** [comment]
I would tend to use the data structures from the instsller in the Update script, too, and move the 'drop table if exists' lines into a separate one which is only executed by the installer before creating the tables.

**Comment by mrbaseman** [comment]
I have created #409 for this change.

I have decided to create a pull request because it is a fundamental change of the update mechanism and I would like to have it tested it - especially in the context of our recent case with the dbsessions table. I'll try to reproduce the scenario of that case and see if 2a51cebca0f4d315ee826c0f858d8f1e7958f083 can avoid such situations. 

Note that I haven't started testing yet, so I might commit some bugfix commits iin the near future, but I think a PR is a nice tool to involve the whole team in the testing phase.
